### PR TITLE
Add details about the debugging of the driver

### PR DESCRIPTION
**Pull request by humblec**
This commit adds details about how to debug, update and watch
logs of iscsi csi driver.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

> /kind documentation
-->
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,87 @@
...
+csi-iscsi-node-klh5c                         3/3     Running   0          10s
 ```
 
-#### NodeUnpublish a volume
+#### Get details about the iscsi driver deployed
+
 ```
-$ csc node unpublish --endpoint tcp://127.0.0.1:10000 --target-path /mnt/iscsi iscsitestvol
-iscsitestvol
+[#]$ kubectl get csidriver
```
Comment (andyzhangx) [inline README.md:23]: that's too more details, not appropriate in readme, it's better add this as separate debug doc?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,87 @@
...
+Events:  <none>
 ```
 
-#### Get NodeID
+#### Watch logs of node plugin container as shown below
+
 ```
-$ csc node get-id --endpoint tcp://127.0.0.1:10000
-CSINode
+[#]$ kubectl logs -f csi-iscsi-node-klh5c -c iscsi
```
Comment (andyzhangx) [inline README.md:72]: and this, too details, better add to a separate debug doc

File: README.md (hunk 1 above)
Comment (humblec) [reply README.md:23]: yeah, may be I can remove the describe output from here and only keep get csidriver info output @andyzhangx .. wdyt?

File: README.md (hunk 2 above)
Comment (humblec) [reply README.md:72]: The only thing which I want to get rid here is the csc examples which are pretty old or not a proper thing to have in a repo atleast in this stage..

**Comment by humblec** [comment]
@andyzhangx added debug doc, rest of the information I will populate in followup PRs.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**